apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -2,6 +2,7 @@ import { CalendarIcon, ClockIcon, CreditCardIcon, ExclamationIcon } from "@heroi
 import { EventTypeCustomInputType } from "@prisma/client";
 import { useContracts } from "contexts/contractsContext";
 import dayjs from "dayjs";
+import { useSession } from "next-auth/react";
```

Comment:
Note: Here we try the new useSession hook from next-auth, we don't use our current useMe query used elsewhere on the codebase.